src/scrabble.js

```diff
@@ -1,8 +1,12 @@
 const { LETTERSCORES, SCOREMULTIPLIER } = require('./constants')
```

Comment:
I would be interested to hear how you decided to go about structuring this.

Reply:
Yeah of course! I started off with detailing what I wanted to do in my [pseudocode](https://github.com/Uprising5034/js-scrabble-challenge/blob/main/pseudocode.md) then it was a case of converting that pseudocode I planned into Javascript.

Essentially I break down the problem into small tasks which I then later made into functions, then identify what input they need and what they should be outputting.